.github/workflows/dummy.yml

```diff
@@ -0,0 +1,30 @@
+name: Add a Dummy commit to trigger build
+
+on:
+  push:
+    branches: master
```

Comment:
change this temporarily to this branch name and schedule a dummy commit frequency for test purpose. You can push the dummy commits to a dummy branch for testing.

Reply:
it only works when workflow is committed to master